src/me.rs

```diff
@@ -116,6 +127,13 @@ pub fn get_sad(
 
   sum
 }
+}
+
+#[cfg(all(target_arch = "x86_64", not(windows), feature = "nasm"))]
+pub use self::nasm::get_sad;
+
+#[cfg(not(any(target_arch = "x86_64", not(windows), feature = "nasm")))]
```

Comment:
I'm pretty sure this is why appveyor fails on a windows build.

Reply:
You are absolutely right :) Thank you for spotting it.